.github/workflows/regression-workflow-instance-join.yml

```diff
@@ -0,0 +1,84 @@
+name: Workflow Instance Join
+
+on:
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      # Matrix paths are independent, and failure for one provider should not
+      # halt the running jobs testing another provider.
+      fail-fast: false
+      # See matrix of secrets...
+      # https://sbulav.github.io/terraform/github-actions-matrix-secrets/
+      matrix:
+        include:
```

Comment:
What does everyone think about using [Alice/Bob](https://en.wikipedia.org/wiki/Alice_and_Bob) actor naming? I've started with that in #148 (.github/workflows/report-exchange.yml) with the goal of updating that matrix to various different permutations of actors.

It probably also makes sense to use full secret sets (even with some value duplication) for all of these instead of sharing secrets between orgs (e.g., re-use of `TRANSMUTE_PRODUCTION_TOKEN_ENDPOINT` for orgs 1, 2, and 3) to facilitate actor switching via matrix.